posix/posix.c

```diff
@@ -1993,14 +1993,45 @@ int posix_setsockopt(int socket, int level, int optname, const void *optval, soc
 int posix_utimes(const char *filename, const struct timeval *times)
 {
 	oid_t oid;
-
 	if (proc_lookup(filename, NULL, &oid) < 0)
 		return -ENOENT;
 
 	return 0;
 }
 
 
+int posix_futimes(int fildes, const struct timeval *times)
+{
+	TRACE("futimes(%d)", fildes);
+
+	open_file_t *f;
+	msg_t msg;
+	int err;
+
+	if ((err = posix_getOpenFile(fildes, &f)) < 0)
+		return err;
```

Comment:
maybe we should guard against `times` being null (somebody calling syscall directly) and return error (`futimes(_, NULL)` function call handled correctly in libphoenix)

Reply:
We can assume that this pointer should not be NULL.

Reply:
@nalajcie We should really sanitize all pointers passed to the kernel - not only check if it's not NULL but also if it belongs to the caller's map

Reply:
Agreed but this applies to all syscalls so we should have some common sanitizing code.